src/test/java/com/bravo/user/service/AddressServiceTest.java

```diff
@@ -0,0 +1,79 @@
+package com.bravo.user.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import com.bravo.user.App;
+import com.bravo.user.dao.model.Address;
+import com.bravo.user.dao.model.mapper.ResourceMapper;
+import com.bravo.user.dao.repository.AddressRepository;
+import com.bravo.user.model.dto.AddressDto;
+
+@ContextConfiguration(classes = { App.class })
+@ExtendWith(SpringExtension.class)
+@SpringBootTest
+class AddressServiceTest {
+
+	@Autowired
+	private AddressService addressService;
+
+	@MockBean
+	private ResourceMapper resourceMapper;
+
+	@MockBean
+	private AddressRepository addressRepository;
+
+	private List<AddressDto> dtoAddresses;
+
+	@BeforeEach
+	public void beforeEach() {
+		final List<Integer> ids = IntStream.range(1, 10).boxed().collect(Collectors.toList());
```

Comment:
I like streams.  This falls as a non-judicious use IMHO https://dev.to/kylec32/effective-java-use-stream-judiciously-4178.

Reply:
My goal was to remain as similar to the development and test approach to the User components (which is using this already). In order to adjust, I'm curious what are your thoughts though on why this is non-judicious? Is it not explicit enough or should it just be handled in a more straightforward way?

Would you recommend I hardcode a list of ids to test with instead?

Reply:
> Would you recommend I hardcode a list of ids to test with instead?

Was thinking about this too.  A simple for loop would work here with a list.add.  The end result is to get the same List of integers and this code does that before every test.  This could be run once and used for all tests that way. 

I googled it to learn this one myself :) https://stackoverflow.com/questions/10242380/how-can-i-generate-a-list-or-array-of-sequential-integers-in-java  has variations with performance implications.  In this case, I think both of the lines could be combined into IntStream.rangeClosed(1, 10).forEach(x -> logic)  where the logic in the next line would be  within the stream.  That has the benefit of the range not being stored in a collection which is a bit better on memory.  

The code in the codebase is sometimes not perfect :) .   One of the goals is to see things that you pick up that we miss.  You did this in the test code which was awesome!

TLDR instead of stream -> List -> stream  go with just stream for the logic in lines 46-49.